lib/generate_models.rb

```diff
@@ -222,6 +224,9 @@
     contents.gsub!(/  field :qdm_status, type: String\n/, '') # Don't include this field
   end
 
+  # Add class
+  contents.gsub!(/  field :class_name, type: String\n/, "  field :class_name, type: String, default: '#{dc_name.camelize}'\n")
```

Comment:
What's the value behind storing these in the database, as opposed to having them in constants?

Reply:
No huge reason, mainly this variable is used to quickly determine what QDM datatype we are seeing when looking at a patient in JSON format. This field may also come in handy for serialization. If ever a QDM datatype is renamed, we can convert this object to the new type and update this field to whatever the new class name for that datatype is.